scripts/logic.js

```diff
@@ -164,15 +164,17 @@ export function drawAboutFaceIndicator(token) {
 	try {
 		//get the rotation of the token
 		let tokenDirection = token.document.flags[MODULE_ID]?.direction ?? getIndicatorDirection(token.document) ?? 90;
-
+	
+		// Calculate token's scale
+		const tokenScale = (Math.abs(token.document.texture.scaleX) + Math.abs(token.document.texture.scaleY)) / 2;
+	
 		// Calculate indicator's distance
 		const indicatorDistance = game.aboutFace.indicatorDistance;
 		const maxTokenSize = Math.max(token.w, token.h);
-		const distance = (maxTokenSize / 2) * indicatorDistance;
-
+		const distance = (maxTokenSize / 2) * indicatorDistance * tokenScale;
+	
 		// Calculate indicator's scale
 		const tokenSize = Math.max(token.document.width, token.document.height);
-		const tokenScale = Math.abs(token.document.texture.scaleX) + Math.abs(token.document.texture.scaleY);
```

Comment:
The distance is waaaaaay to far. Move `tokenScale` back to where it was and add a `const maxScale = Math.max(token.document.texture.scaleX, token.document.texture.scaleY);` and use that instead.

I suggest you try something like `(maxTokenSize / 2) * indicatorDistance * Math.max(0.2, maxScale / indicatorDistance)` so it takes the user's setting into account, it is 1:1 at Scale 2, 1.5:1 at Scale 3, and doesn't look bad on smaller scales.